qiskit/test/mock/fake_backend_v2.py

```diff
@@ -176,3 +180,40 @@ def qubit_properties(self, qubit):
         if isinstance(qubit, int):
             return self._qubit_properties[qubit]
         return [self._qubit_properties[i] for i in qubit]
+
+
+class FakeBackendSimple(BackendV2):
```

Comment:
Is this class a temporary solution for while `FakeBackendV2` doesn't have `.run` method implemented? What prevents us to implemented `.run` method to `FakeBackendV2` now? Is it blocked by #7391 that I am working on?

Reply:
Yeah, this is just to have something until #7391 is implemented. But also because the goal of this PR is to fix support I wanted to have something smaller to backport than what #7391 will end up being. To test the quantum instance works with BackendV2 there needs to be a backend implementing the v2 interface with `run()` defined. So I threw this together so we can have a minimal backend that does this.